gazebo_ros2_control_demos/config/holonomic_drive_controller.yaml

```diff
@@ -15,13 +15,16 @@ mecanum_drive_controller:
 
     publish_rate: 30.0
 
-    command_joint_names: ["front_left_joint", "rear_left_joint", "rear_right_joint", "front_right_joint"]
+    front_left_wheel_command_joint_name: "front_left_joint"
+    front_right_wheel_command_joint_name: "front_right_joint"
+    rear_right_wheel_command_joint_name: "rear_right_joint"
+    rear_left_wheel_command_joint_name: "rear_left_joint"
```

Comment:
These lines are causing the following error and preventing gazebo from launching successfully:
```
[ERROR] [gzserver-2]: process has died [pid 13488, exit code -11, cmd 'gzserver -slibgazebo_ros_init.so -slibgazebo_ros_factory.so -slibgazebo_ros_force_system.so'].

```